cmd/mod.go

```diff
@@ -256,3 +259,14 @@ func newInstallOpts(cmd *cobra.Command, args ...string) *modinstaller.InstallOpt
 	}
 	return opts
 }
+
+// Modifies(trims) the URL if contains http ot https in arguments
+
+func trimGitUrls(opts *modinstaller.InstallOpts) {
+	re := regexp.MustCompile(`^(https?://)`)
```

Comment:
@Mr-Destructive can we solve this with a `strings.HasPrefix` instead of a `regex` match (which is inherently greedy and expensive)